test/integration/common/TypeDBRunners.kt

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2022 Vaticle
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+package com.vaticle.typedb.studio.test.integration.common
+
+import com.vaticle.typedb.common.test.TypeDBRunner
+import com.vaticle.typedb.common.test.core.TypeDBCoreRunner
+import kotlinx.coroutines.runBlocking
+
+object TypeDBRunners {
+    fun withTypeDB(testFunction: (TypeDBRunner) -> Unit) {
```

Comment:
We now pass the runner itself to the tests for greater flexibility. This makes developing tests that involve any shutdowns/restarts of TypeDB possible.